frontend/src/components/ResourceInfo.tsx

```diff
@@ -71,7 +71,16 @@ export class ResourceInfo extends React.Component<ResourceInfoProps, {}> {
     const customPropertyMap = resource.getCustomPropertiesMap();
     return (
       <section>
-        <h1 className={commonCss.header}>Type: {this.props.typeName}</h1>
+        <h1 className={commonCss.header}>
+          {this.props.typeName}
+          {(() => {
+            const stateText = getResourceStateText(this.props);
+            if (stateText) {
+              return ` (${stateText})`;
+            }
+            return '';
+          })()}
+        </h1>
```

Comment:
because we didn't log state before, if we found state=Unknown, it's better not to render it

Reply:
The text change from `Type: <type-name>` to `<type-name> (<state>)` is following principle that, when text content can tell what it is, UI would feel more concise when the `Type:` and `State:` prefix is omitted.

For example, when we see `kfp.ContainerExecution`, we know it is a type of execution, so it's unnecessary to add prefix `Type: ` now.